modules/test/playwright/tests/dynamic-data-mapping-form-web/main/formSubmission.spec.ts

```diff
@@ -169,4 +170,48 @@ test.describe('Manage forms through submission page', () => {
 
 		await expect(page.getByText('123456')).toBeVisible();
 	});
+
+	test('verify that a Form can require CAPTCHA before being accessed', async ({
+		formBuilderPage,
+		formsPage,
+		page,
+	}) => {
+		await formsPage.goTo();
+
+		await test.step('create a form containing a text field and CAPTCHA validation', async () => {
+			await formsPage.newFormButton.first().click();
+
+			await formBuilderPage.fillFormTitle('Form' + getRandomInt());
+
+			await page
+				.getByRole('button', {name: 'Text field'})
+				.first()
+				.dblclick();
+
+			await page.getByRole('button', {name: 'settings'}).click();
+
+			await page.getByLabel('Require CAPTCHA').click();
+
+			await page.getByRole('button', {name: 'Done'}).click();
+
+			await formBuilderPage.clickPublishFormButton();
+		});
+
+		await test.step('navigate to the form page and assert that CAPTCHA is reiquired', async () => {
+			const formSubmissionURL =
+				await formBuilderPage.getFormSubmissionURL();
+
+			await page.goto(formSubmissionURL, {waitUntil: 'networkidle'});
+
+			await page.getByLabel('Text').fill('Text field value');
+
+			await page.getByRole('textbox').last().fill('1');
```

Comment:
Hi @DavysonMelo, could you please review the "'make sure the aria-labelledby reference is present in the captcha form view'" test and see if we can locate the captcha field using the "captcha" label. By doing so, we could remove the mentioned test.